.gitignore

```diff
@@ -13,6 +13,7 @@ mattermost-webapp.iml
 junit.xml
 coverage
 config.override.mk
+*.tsbuildinfo
```

Comment:
These files contain information used by the TypeScript compiler to build multiple packages. I think it's things like which files depend on what so that it can cache compiled code